stampery-trailbot-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,211 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "Username for the Virtual Machine."
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for the Virtual Machine."
+      }
+    },
+    "adminEmail": {
+      "type": "string",
+      "metadata": {
+        "description": "Your email address."
+      }
+    },
+    "dnsLabelPrefix": {
```

Comment:
By default, Traibot Client shows the hostname as the identifier for each Trailbot Watcher (in this case, each Secure Ubuntu server).
This parameter is supplied in order to allow the user to set a friendly name for the server, so the Client shows something like "My web server" instead of "ip-172-31-7-171" or similar.